modules/remotecontrol/Receiver.js

```diff
@@ -100,9 +100,7 @@ export default class Receiver extends RemoteControlParticipant {
             this._userLeftListener);
         transport.sendEvent({
             name: REMOTE_CONTROL_EVENT_TYPE,
```

Comment:
One more thing that bothers me while I'm looking at this now is that the actual "name" is called XXX_EVENT_TYPE. Should probably be XXX_EVENT_NAME.